options/linux/include/syscall.h

```diff
@@ -0,0 +1 @@
+#include <sys/syscall.h>
```

Comment:
Why is this needed? This should probably live in the sysdeps folder since it's specific to the Linux kernel.

Reply:
This header is present on both glibc and musl, and Dennis ran into some program expecting this to exist. Both of the other libcs just include `<sys/syscall.h>` as well.

Reply:
Ok, but it should still probably go in the Linux sysdeps. I can conceivably imagine programs checking has_include for this header and making decisions based on that, so it's better for it to not be present unless it's implemented correctly.

Reply:
Isn't sys/syscall.h also installed as a part of option/linux?

Reply:
There's one there, but I don't think it's used and we ought to remove it.

Reply:
hm? what do you mean?

Reply:
Check options/linux/include/sys/syscall.h and sysdeps/linux/include/sys/syscall.h